lib/hackmode/p2p-backend-hackmode-peer.js

```diff
@@ -28,6 +35,52 @@ const { DhtBackendFullsync } = require('../n3h-mod-dht-fullsync')
  * @param {string} [options.wssRelayPeer] - if set, advertise this node as relaying through a specific peer target at this peerTransport address. must be a direct address, not another relay.
  */
 class P2pBackendHackmodePeer extends AsyncClass {
+  /**
+   * @param workDir - Directory where to search for the keyfile
+   * @returns {keyBundle} KeyBundle loaded from disk or randomly generated and saved to disk
+   */
+  async loadKey (workDir) {
+    util.pwhashOpslimit = mosodium.pwhash.OPSLIMIT_INTERACTIVE
+    util.pwhashMemlimit = mosodium.pwhash.MEMLIMIT_INTERACTIVE
```

Comment:
^^ I think we should leave the default at SENSITIVE, and set INTERACTIVE only for unit tests

Reply:
Agreed, but I haven't managed to get it to work with that setting, and the passphrase is hardcoded anyway...